doc/release/platforms/README.sgi.rst

```diff
@@ -49,8 +55,7 @@ This can be done using the following settings:
 
```

Comment:
I mean, if we know there's something in here that is wrong and misleading, why are we putting it on the web, even with a warning?